packages/salesforcedx-vscode-lwc/src/channel/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ChannelService } from '@salesforce/salesforcedx-utils-vscode/out/src/commands';
```

Comment:
@xyc I ended up doing it this way instead of having it as part of the extension activation since this allows us to initialize the channel service when the extension code is loaded which happens before activation.

Reply:
Sounds good this works too.